transformer_engine/paddle/fp8_buffer.py

```diff
@@ -179,14 +202,17 @@ def wait(self) -> None:
     def to_numpy(self) -> Dict[str, List[np.array]]:
         """Convert to numpy arrays"""
         out = {}
-        for k, v in self._data.items():
+        for k, v in self._global_amax.items():
             out[k] = [tensor.numpy() for tensor in v]
         return out
 
     def from_numpy(self, buffer: Dict[str, np.array]) -> None:
         """Set buffer values from numpy arrays"""
         for k, v in buffer.items():
-            self._data[k] = [paddle.to_tensor(arr) for arr in v]
+            self._global_amax[k] = [paddle.to_tensor(arr) for arr in v]
+
+    def enable_cudagraph(self):
+        self._use_cudagraph = True
```

Comment:
Should we support the case where we disable CUDA graphs?
```suggestion
    def enable_cudagraph(self, enable: bool = True) -> None:
        self._use_cudagraph = enable
```
For example, a user might want to pretrain with CUDA graphs and do finetuning without. This may be something to defer until later if it will require significant work.

Reply:
From my past experience, enabling cudaGraph and then disabling it causes unexpected segmentation faults. We will investigate this issue further in the future if this feature is truly needed. For now, we should maintain the enable-only approach.